examples/vector.json

```diff
@@ -0,0 +1,174 @@
+{
+  "stac_version": "1.0.0",
+  "stac_extensions": [
+    "https://stac-extensions.github.io/datacube/v2.1.0/schema.json"
+  ],
+  "id": "vector-datacube",
+  "type": "Feature",
+  "geometry": {
+    "type": "MultiPolygon",
+    "coordinates": [
+      [
+        [
+          [
+            -122.40863800048828,
+            37.60280825923639
+          ],
+          [
+            -122.35301971435545,
+            37.60280825923639
+          ],
+          [
+            -122.35301971435545,
+            37.641694203264265
+          ],
+          [
+            -122.40863800048828,
+            37.641694203264265
+          ],
+          [
+            -122.40863800048828,
+            37.60280825923639
+          ]
+        ]
+      ],
+      [
+        [
+          [
+            -122.51575469970703,
+            37.50373410549169
+          ],
+          [
+            -122.48313903808594,
+            37.50373410549169
+          ],
+          [
+            -122.48313903808594,
+            37.52443079581378
+          ],
+          [
+            -122.51575469970703,
+            37.52443079581378
+          ],
+          [
+            -122.51575469970703,
+            37.50373410549169
+          ]
+        ]
+      ],
+      [
+        [
+          [
+            -122.26203918457031,
+            37.69726847113855
+          ],
+          [
+            -122.19577789306639,
+            37.69726847113855
+          ],
+          [
+            -122.19577789306639,
+            37.747507619675595
+          ],
+          [
+            -122.26203918457031,
+            37.747507619675595
+          ],
+          [
+            -122.26203918457031,
+            37.69726847113855
+          ]
+        ]
+      ],
+      [
+        [
+          [
+            -122.13415145874025,
+            37.65175228874552
+          ],
+          [
+            -122.10994720458984,
+            37.65175228874552
+          ],
+          [
+            -122.10994720458984,
+            37.66765215806202
+          ],
+          [
+            -122.13415145874025,
+            37.66765215806202
+          ],
+          [
+            -122.13415145874025,
+            37.65175228874552
+          ]
+        ]
+      ]
+    ]
+  },
+  "bbox": [
+    -122.51575469970703,
+    37.50373410549169,
+    -122.10994720458984,
+    37.747507619675595
+  ],
+  "properties": {
+    "title": "Multi-dimensional vector data cube in a STAC Item.",
+    "datetime": "2016-05-03T13:21:30.040Z",
```

Comment:
How is this related with the values in cube:dimensions -> "t" -> "temporal" ?

Reply:
Good point, I updated the example to include start and end datetimes.